roles/openshift_cloud_provider/tasks/aws.yml

```diff
@@ -1,4 +1,13 @@
-- name: Create cloud config
+# Work around ini_file create option in 2.2 which defaults to no
+- name: Create cloud config file
+  file:
+    dest: "{{ openshift.common.config_base }}/cloudprovider/aws.conf"
+    state: touch
```

Comment:
I double checked the docs to be sure about this one

> If `touch` (new in 1.4), an empty file will be created if the path does not exist, while an existing file or directory will receive updated file access and modification times (similar to the way `touch` works from the command line).

So in effect this means our playbook will always say something `changed`. I know we have a lot of tasks that do that already, but IMHO that's misleading behavior and we should avoid it. What about adding

```yaml
- name: Create cloud config file
  ...
  changed_when: false
```

Since this is just a work-around for the missing `create` option in older versions?